Automattic/wp-calypso
# Help Center: Use correct colors for redesigned help center

Related to https://github.com/Automattic/dotcom-forge/issues/9763
Fixes https://github.com/Automattic/dotcom-forge/issues/9763

## Proposed Changes

* Use Help Center redesigned colors instead themes color

Before:
<img width="418" alt="image" src="https://github.com/user-attachments/assets/a595744d-295a-4f0b-b139-6af396f58203">


After:

<img width="417" alt="image" src="https://github.com/user-attachments/assets/e5ce5207-96b0-4365-ae6f-a35e694ac000">


## Why are these changes being made?


* Keep Help Center style persistent.

## Testing Instructions



* Use live link and add the flag `?flags=help-center-experience`
* Change your sites theme.
* Check the Help Center buttons, hover and focus, they should be the same in all themes
* Check issue for more reference https://github.com/Automattic/dotcom-forge/issues/9763

## Pre-merge Checklist
